packages/connect-popup/e2e/tests/methods.test.ts

```diff
@@ -7,11 +7,16 @@ import { TrezorUserEnvLink } from '@trezor/trezor-user-env-link';
 import { fixtures } from './__fixtures__/methods';
 import { buildOverview } from '../support/buildOverview';
 import { ensureDirectoryExists } from '@trezor/node-utils';
-import { log } from '../support/helpers';
+import { getExtensionPage, log } from '../support/helpers';
 
 const url = process.env.URL || 'http://localhost:8088/';
 const emuScreenshots: Record<string, string> = {};
 
+let device = {};
+let context: any = null;
+
+const isExtension = process.env.IS_WEBEXTENSION === 'true';
```

Comment:
just thinking. coulnd't this be abstractad away to support/helpers? so that the person that would be writing tests doesnt have to think about implementation details? This does not have high priority, only if you get some idea..